ods_ci/tests/Tests/0200__rhoai_upgrade/0201__pre_upgrade.robot

```diff
@@ -246,8 +249,44 @@ Model Registry Pre Upgrade Set Up
     [Tags]      Upgrade     ModelRegistryUpgrade
     Model Registry Pre Upgrade Scenario
 
+Long Running Jupyter Notebook
+    [Documentation]    Launch a long running notebook before the upgrade
+    [Tags]      Upgrade
+    Launch Notebook
+    Add And Run JupyterLab Code Cell In Active Notebook     ${CODE}
+
+    # Get the notebook pod creation timestamp
+    ${notebook_pod_name}=    Set Variable    jupyter-nb-ldap-2dadmin2-0
```

Comment:
why not getting the nb name using either `Get User CR Notebook Name` or `Get User Notebook Pod Name`
?

Reply:
good point, it was just copy-paste thing; I'll update that as you proposed :+1:

Reply:
rebased and implemented, thanks!